src/main/java/com/szymon/repository/BookRepositoryInMemory.java

```diff
@@ -0,0 +1,47 @@
+package com.szymon.repository;
+
+import com.szymon.domain.Book;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by szymon.nowak on 04.04.2016.
+ */
+@Component
+public class BookRepositoryInMemory implements BookRepository {
+
+    private List<Book> bookList = new ArrayList<Book>();
+
+
+    public Book findOne(int id) {
```

Comment:
I think find method should have different parameter than internal list id. Title, author? How to enable both?